script/shared/github.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from "fs";
+import path from "path";
+import decompress from "decompress";
+
+import { downloadFile } from "./utils";
+
+const RELEASE_API_URL =
+  "https://api.github.com/repos/many-things/cw-hyperlane/releases";
```

Comment:
Can you move the repo name as a separate constant so that it is easier to override in forks?

Reply:
Sure, applying.